gtpv1/ie/extended-common-flags.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2024 go-gtp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package ie
+
+import "io"
+
+// NewExtendedCommonFlags creates a new ExtendedCommonFlags IE.
+//
+// Note: each flag should be set in 1 or 0.
+func NewExtendedCommonFlags(uasi, bdwi, pcri, vb, retloc, cpsr, ccrsi, unauthenticatedIMSI int) *IE {
```

Comment:
May be params should be `uint8`?

Reply:
Made int to be consistent with `NewCommonFlags` function that uses int